GithubProfileWiki/Networking/APIHandler.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  APIHandler.swift
+//  GithubProfileWiki
+//
+//  Created by ilter on 23.01.2022.
+//
+
+import Foundation
+
+public enum HTTPMethod: String {
+    case GET
+    case POST
+    case PUT
+    case PATCH
+    case DELETE
```

Comment:
all capital letters seems oldskull

Reply:
Yeah I wrote like this for to use rawValue but will change that looks like Java :D